README.md

```diff
@@ -246,6 +246,17 @@ At the time of writing, there is only one host example available:
 
 - tinyusb_host_cdc_msc_hid
 
+### USB Dual Mode
+
+USB Dual Mode uses PIO as a USB host controller and the RP2040 USB device controller as a device controller. All the USB dual examples come directly from the TinyUSB dual examples directory [here](https://github.com/hathach/tinyusb/tree/master/examples/dual).
+Those that are supported on RP2040 devices are automatically included as part of the pico-examples
```

Comment:
Just a nitpick, but if USB Dual Mode uses PIO, would there ever be any examples that _aren't_ supported on RP2040? :wink: